apps/lottery/src/utils/index.ts

```diff
@@ -0,0 +1,19 @@
+import { Hex } from '@gear-js/api';
+import type { InjectedAccountWithMeta } from '@polkadot/extension-inject/types';
+import { LOCAL_STORAGE } from 'consts';
+import { getStatus, getCountdown } from './status';
+
+const isLoggedIn = ({ address }: InjectedAccountWithMeta) => localStorage[LOCAL_STORAGE.ACCOUNT] === address;
+
+const isHex = (value: unknown) => {
+  const hexRegex = /^0x[\da-fA-F]+/;
+
+  return typeof value === 'string' && hexRegex.test(value);
+};
```

Comment:
You can use `isHex` function from `@polkadot/util` package

Reply:
Yep, but I don't feel the urge to install entire lib just for one function

Reply:
it already installed, when you installed @polkadot/api